site/content/3.12/index-and-search/analyzers.md

```diff
@@ -117,7 +117,7 @@ against other databases by specifying the prefixed name, e.g.
 The following Analyzer types are available:
 
 - [`identity`](#identity): treats value as atom (no transformation)
-- [`delimiter`](#delimiter): splits into tokens at user-defined character
+- [`delimiter`](#delimiter): splits into tokens at user-defined characters
```

Comment:
New analyzer will be separate type, because single delimiter analyzer support escaping delimiters (it's probably should be named csv analyzer, you can check rfc that mentioned in the it's docs)